app/views/govuk_publishing_components/components/_layout_super_navigation_header.html.erb

```diff
@@ -68,9 +106,11 @@
             <!--[if IE 8]>
             <img src="<%= asset_path('govuk-logotype-crown.png') %>" alt="" class="govuk-header__logotype-crown-fallback-image" width="36" height="32">
             <![endif]-->
-            <span class="govuk-header__logotype-text">
-              <%= logo_text %>
-            </span>
+            <% unless hide_logo_text %>
```

Comment:
Since we're removing the link text, I think we should add an `aria-label` to the link. It's got a `title` but I think `aria-label` tests better for accessibility eg. https://www.deque.com/blog/text-links-practices-screen-readers/. Worth researching at least. Another alternative might be including visually hidden link text.

Reply:
This works well on NVDA and JAWS on Mac desktop but unfortunately Mac Voiceover doesn't read the label and skips the link entirely 😞 I think we should go with visually hidden text instead.

Reply:
Just realised that Mac Safari skips the link too (with or without Voiceover enabled).

Reply:
(Continued conversation in https://github.com/alphagov/govuk_publishing_components/pull/3566/files#r1345700075 as this current conversation was getting lost as it's so old)